docs/standard/base-types/custom-numeric-format-strings.md

```diff
@@ -191,23 +195,55 @@ You can create a custom numeric format string, which consists of one or more cus
  The following example uses the ";" format specifier to format positive, negative, and zero numbers differently.  
   
  [!code-cpp[Formatting.Numeric.Custom#8](../../../samples/snippets/cpp/VS_Snippets_CLR/formatting.numeric.custom/cpp/custom.cpp#8)]
- [!code-csharp[Formatting.Numeric.Custom#8](../../../samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/custom.cs#8)]
+ [!code-csharp-interactive[Formatting.Numeric.Custom#8](../../../samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/custom.cs#8)]
  [!code-vb[Formatting.Numeric.Custom#8](../../../samples/snippets/visualbasic/VS_Snippets_CLR/formatting.numeric.custom/vb/Custom.vb#8)]  
   
  [Back to table](#table)  
-  
+
+## Character literals  
+ 
+The following characters in a custom numeric format string are reserved and are always interpreted as formatting characters:  
+
+- 0
+- \#
+- %
+- ‰
+- '
+- \
+- .
+- ,
+- E or e, depending on its position in the format string.
```

Comment:
I'm not sure this list should be here, as the whole topic and the table at the start are about those characters. If the list makes it here anyway, what about the following idea: add the link to corresponding section in the topic for each format specifier?

Reply:
I kept the list but added the link to the corresponding format specifier, @pkulikov.